src/core_modules/capture-core/components/Pages/EnrollmentAddEvent/EnrollmentAddEventPage.container.js

```diff
@@ -1,126 +1,74 @@
+        return EnrollmentAddEventPageStatuses.DEFAULT;
+    }, [enrollmentId, pageIsInvalid, programId, teiId, validIds]);
 
-    const rulesExecutionDependencies = useMemo(() => ({
-        events: enrollment?.events,
-        attributeValues,
-        enrollmentData: {
-            enrollmentDate: enrollment?.enrollmentDate,
-            incidentDate: enrollment?.incidentDate,
-            enrollmentId: enrollment?.enrollment,
-        },
-    }), [enrollment, attributeValues]);
-
-    const {
-        handleSetOrgUnitId,
-        handleResetOrgUnitId,
-        handleResetProgramId,
-        handleResetEnrollmentId,
-        handleResetTeiId,
-        handleResetStageId,
-        handleResetEventId,
-        teiDisplayName,
-        trackedEntityName,
-        enrollmentsAsOptions,
-        teiSelectorFailure,
-        userInteractionInProgress,
-    } = useEnrollmentAddEventTopBar(teiId, programId, enrollment);
-
-    return (
-        <>
-            <EnrollmentAddEventTopBar
-                programId={programId}
-                orgUnitId={orgUnitId}
-                enrollmentId={enrollmentId}
-                teiDisplayName={teiDisplayName}
-                trackedEntityName={trackedEntityName}
-                stageName={selectedProgramStage?.stageForm.name}
-                eventDateLabel={selectedProgramStage?.stageForm.getLabel('eventDate')}
-                enrollmentsAsOptions={enrollmentsAsOptions}
-                onSetOrgUnitId={handleSetOrgUnitId}
-                onResetOrgUnitId={handleResetOrgUnitId}
-                onResetProgramId={handleResetProgramId}
-                onResetEnrollmentId={handleResetEnrollmentId}
-                onResetTeiId={handleResetTeiId}
-                onResetStageId={handleResetStageId}
-                onResetEventId={handleResetEventId}
-                userInteractionInProgress={userInteractionInProgress}
-                teiSelectorFailure={teiSelectorFailure}
-                enrollmentSelectorFailure={Boolean(commonDataError)}
-            />
-            <EnrollmentAddEventPageComponent
-                programId={programId}
-                stageId={stageId}
-                orgUnitId={orgUnitId}
-                teiId={teiId}
-                enrollmentId={enrollmentId}
-                onSave={handleSave}
-                onCancel={handleCancel}
-                onDelete={handleDelete}
-                widgetEffects={outputEffects}
-                hideWidgets={hideWidgets}
-                widgetReducerName={widgetReducerName}
-                rulesExecutionDependencies={rulesExecutionDependencies}
-                pageFailure={Boolean(commonDataError)}
-                ready={Boolean(enrollment)}
-                dataEntryHasChanges={dataEntryHasChanges}
+    switch (pageStatus) {
+    case EnrollmentAddEventPageStatuses.LOADING:
+        return <LoadingMaskForPage />;
+    case EnrollmentAddEventPageStatuses.MISSING_REQUIRED_VALUES:
```

Comment:
Hey, 
Do you think it makes sense to use the [NoticeBox](https://ui.dhis2.nu/demo/?path=/story/data-display-notice-box--error) component to display all these warnings messages? 
Thanks!